exercises/transpose/example.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+function transpose(text) {
+  return text.split('\n').reduce(function (result, line, lineNo) {
+    line.split('').map(function (value, key) {
+      if (typeof result[key] === 'undefined') {
+        result[key] = new Array(lineNo + 1).join( ' ' );
+      }
+      result[key] += value;
+    });
+    return result;
+  }, []).join('\n');
+}
+
```

Comment:
Can you please run ESLint against your PR?  There are a few small style issues with the code, and I don't want to waste your time calling them out individually.  But if you use the linter it can fix your code automatically!

You can lint a _single_ exercise dir like this:
1. Make sure you're in the root level of the javascript repo
2. Make sure you've run `npm install`
3. run `./node_modules/.bin/eslint exercises/transpose/ --fix`

This will fix any small style issues (EG whitespace, braces, etc) that it can, and display errors for anything it cannot fix.  Big time-saver.